com.microsoft.mrtk.input/Visualizers/RiggedHandVisualizer/RiggedHand-Main.mat

```diff
@@ -2,14 +2,16 @@
 %TAG !u! tag:unity3d.com,2011:
 --- !u!21 &2100000
 Material:
-  serializedVersion: 6
+  serializedVersion: 8
```

Comment:
Is this new serialization from Unity 2020 or 2021? If 2021, is 2020 okay with it? 😬

Reply:
2020 can't deserialize 2021.x material keywords. (It just silently fails.) 😢 

In Graphics Tools we have to ensure content is authored in 2020.

Reply:
Let me check and reserialize. I was doing some testing in 2021, this might have been accidentally borked

Reply:
Reserialized to 2020.x, serializedVersion=6.